packages/spec/integration/create-hops-app/__tests__/index.spec.js

```diff
@@ -5,7 +5,7 @@ const { execSync } = require('child_process');
 const createHopsAppBin = require.resolve('create-hops-app');
 
 describe('create-hops-app', () => {
-  const version = 'latest';
+  const version = 'next';
```

Comment:
Should we try to find a way to automatically determine what tag to use here, so that we don't run into the problem again, that we forget to change it after the release?
We could for example:
- look into the package.json and see if the version is a prerelease?
- look into lerna.json, to see if the version is a prerelease?